generate an atom feed

PhilTaken opened the pull request:
Building the email now also generates an atom feed. This does however require adding a date to all issues. For existing issues I determined the publishing date from the git history.

I tested the generated feed with my home-cooked feed reader so it would be best if you test it again before merging.

PhilTaken committed:
generate an atom feed

CrowdHailer commented:
Hi thanks for this. I think I messed up the commits so the dates are probably not correct from there. I have managed to send on every thursday since starting so that's the correct dates. Maybe it doesn't matter so much them being correct into the past.

Would you mind looking into testing with a standard reader and/or finding some online validation tools. I don't use RSS so it'd be good to have a bit more testing

PhilTaken committed:
improve generated feed

in accordance with the official spec https://validator.w3.org/

- use correct date format
- add a type for contained html
- use hrefs for links
- add a self-type link

PhilTaken commented:
Good point, proper validation is key.

Unfortunately, the w3 validator had a few complaints which required a PR to xmleam: https://github.com/tovedetered/xmleam/pull/2

I hope this can be fixed soon :/

PhilTaken commented:
n.b. I used https://validator.w3.org/feed/#validate_by_input to validate the generated xml

CrowdHailer commented:
Great thank you. I will try and get to this this weekend
